src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -12,6 +12,121 @@ namespace System.Tests
+        // InvariantCultureIgnoreCase
+        [InlineData("Hello", "ello", StringComparison.InvariantCultureIgnoreCase, true)]
+        [InlineData("Hello", "ELL", StringComparison.InvariantCultureIgnoreCase, true)]
+        [InlineData("Hello", "ElLo", StringComparison.InvariantCultureIgnoreCase, true)]
+        [InlineData("Hello", "Larger Hello", StringComparison.InvariantCultureIgnoreCase, false)]
+        [InlineData("Hello", "Goodbye", StringComparison.InvariantCultureIgnoreCase, false)]
+        [InlineData("", "", StringComparison.InvariantCultureIgnoreCase, true)]
+        [InlineData("", "hello", StringComparison.InvariantCultureIgnoreCase, false)]
+        [InlineData("Hello", "", StringComparison.InvariantCultureIgnoreCase, true)]
+        [InlineData("Hello", "ell" + SoftHyphen, StringComparison.InvariantCultureIgnoreCase, true)]
+        [InlineData("Hello", "Ell" + SoftHyphen, StringComparison.InvariantCultureIgnoreCase, true)]
+        // Ordinal
+        [InlineData("Hello", "ello", StringComparison.Ordinal, true)]
+        [InlineData("Hello", "ELL", StringComparison.Ordinal, false)]
+        [InlineData("Hello", "ElLo", StringComparison.Ordinal, false)]
+        [InlineData("Hello", "Larger Hello", StringComparison.Ordinal, false)]
+        [InlineData("Hello", "Goodbye", StringComparison.Ordinal, false)]
+        [InlineData("", "", StringComparison.Ordinal, true)]
+        [InlineData("", "hello", StringComparison.Ordinal, false)]
+        [InlineData("Hello", "", StringComparison.Ordinal, true)]
+        [InlineData("Hello", "ell" + SoftHyphen, StringComparison.Ordinal, false)]
+        [InlineData("Hello", "Ell" + SoftHyphen, StringComparison.Ordinal, false)]
+        // OrdinalIgnoreCase
+        [InlineData("Hello", "ello", StringComparison.OrdinalIgnoreCase, true)]
+        [InlineData("Hello", "ELL", StringComparison.OrdinalIgnoreCase, true)]
+        [InlineData("Hello", "ElLo", StringComparison.OrdinalIgnoreCase, true)]
+        [InlineData("Hello", "Larger Hello", StringComparison.OrdinalIgnoreCase, false)]
+        [InlineData("Hello", "Goodbye", StringComparison.OrdinalIgnoreCase, false)]
+        [InlineData("", "", StringComparison.OrdinalIgnoreCase, true)]
+        [InlineData("", "hello", StringComparison.OrdinalIgnoreCase, false)]
+        [InlineData("Hello", "", StringComparison.OrdinalIgnoreCase, true)]
+        [InlineData("Hello", "ell" + SoftHyphen, StringComparison.OrdinalIgnoreCase, false)]
+        [InlineData("Hello", "Ell" + SoftHyphen, StringComparison.OrdinalIgnoreCase, false)]
+        public static void Contains(string s, string value, StringComparison comparisonType, bool expected)
+        {
+            Assert.Equal(expected, s.Contains(value, comparisonType));
+        }
+
+        [Fact]
+        public static void Contains_StringComparison_TurkishI()
+        {
+            string str = "\u0069\u0130";
+            RemoteInvoke((source) =>
```

Comment:
Are there other tests that do the same? (I'm currently on my mobile phone, can't look)